include/mbedtls/config.h

```diff
@@ -2595,4 +2604,6 @@
 
 #include "check_config.h"
 
+#endif /* MBED_OS_MBEDTLS && !DEVICE_ENTROPY_SOURCE */
+
```

Comment:
In general I agree, that this is how this file should look in mbed OS, but I would like it much better if these changes would be achieved by our mbed-os import script:
- These lines don't bring any value to our standalone customers and may be confusing for them. 
- If this would be done by the import script, then we wouldn't need to hack check-names.sh